src/hooks/api.tsx

```diff
@@ -91,22 +117,115 @@ export function useGovernorCouncil(
 export function useProposals(
   governorAddress: string | undefined,
   currentBlock: number | undefined,
-  enabled: boolean = true,
+  enabled: boolean = true
 ): UseQueryResult<Proposal[]> {
   const { network } = useWallet();
-  const paramsDefined = governorAddress !== undefined && currentBlock !== undefined;
+  const paramsDefined =
+    governorAddress !== undefined && currentBlock !== undefined;
 
   async function loadProposals(): Promise<Proposal[]> {
     if (paramsDefined) {
-      const proposals = await fetchProposalsByGovernor(governorAddress);
+      const proposals = (
+        await fetchProposalsByGovernor(governorAddress)
+      ).reverse();
       const updatedProposals: Proposal[] = [];
+      let proposalsToCheck = 5;
```

Comment:
We might be better off sorting this array first, just as an added protection.

We should also just update entries in place, to help with memory usage for long running DAOs.

```js

const proposals = await fetchProposalsByGovernor(governorAddress);
proposals.sort(({ id: a }, { id: b }) => b - a);
```